bugbot/rules/webcompat_score.py

```diff
@@ -0,0 +1,88 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Any, Optional
+
+from bugbot import gcp
+from bugbot.bzcleaner import BzCleaner
+
+
+class WebcompatScore(BzCleaner):
+    def __init__(self):
+        super().__init__()
+        self.autofix_changes = {}
```

Comment:
We do not need to instantiate `autofix_changes` here since this is already the parent class's constructor: https://github.com/mozilla/bugbot/blob/8ede0c2b3a2721c9aa6fe2766fa53146261ef47c/bugbot/bzcleaner.py#L58